app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,20 @@
+<aside>
+  <img src="https://media.giphy.com/media/lddFLE9o2DDGg/giphy.gif" alt= "Rocky running with children">
+  <h4>You've got this!</h4>
+</aside>
+<div id = "task-list">
+  <h2>To Do List</h2>
+
+  <ol>
+    <% @tasks.order(date: :asc).each do |task| %>
+    <li class = "task">
+      <% if task.status == true %>
```

Comment:
The `== true` is redundant here because `if` statements are automatically checking for [truthiness](https://learn.co/lessons/truthiness-in-ruby-readme), and booleans such as `task.status` have "truthiness" that matches their value of `true`/`false`.

Same for line 18 in `show.html.erb`.